draft-ietf-ipsecme-ikev2-sa-ts-payloads-opt.md

```diff
@@ -267,7 +268,12 @@ IKE Session Resumption [RFC5723] defines an IKEv2 extension, that allows peers t
 
 * If support for optimized rekey is negotiated during resumption, then all IKE SA algorithms, including key exchange methods, are taken from the resumption ticket (i.e., from the SA being resumed), since they are not negotiated in the IKE_SA_RESUME exchange.
 
-* The initial Child SA created during the resumption is considered as been created with key exchange methods for the IKE SA, that were stored in the resumption ticket. This is despite the fact, that during the resumption no key exhanges (e.g., Diffie-Hellman) take place, the session keys are derived from the keys stored in the resumption ticket.
+* The initial Child SA created during the resumption is considered as been created with key exchange methods for the IKE SA, that were stored in the resumption ticket. This is despite the fact, that during the resumption no key exchanges (e.g., Diffie-Hellman) take place, the session keys are derived from the keys stored in the resumption ticket.
+
+
+## Mixing Preshared Keys in the CREATE_CHILD_SA Exchanges
+
+{{-qr-alt}} defines how PPKs can be mixed into the session keys calculations. In particular, this document allows PPKs to be used in the CREATE_CHILD_SA exchanges when SAs are being rekeyed. If peers support {{-qr-alt}}, then, in case of optimized rekey, they MUST NOT omit negotiation of the use of PPKs in the CREATE_CHILD_SA exchange.

```

Comment:
Perhaps rephrase: 
If peers support {{-qr-alt}} and a PPK was used for SA being rekeyed, then they MUST NOT silently re-use this PPK in case of optimized rekey without re-negotiation of the use of PPKs in the CREATE_CHILD_SA exchange.